task/aontimer/src/main.rs

```diff
@@ -0,0 +1,75 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#![no_std]
+#![no_main]
+
+use aontimer as aon;
+use drv_riscv_plic_api::*;
+use userlib::*;
+
+task_slot!(INT_CONTROLLER, ext_int_ctrl);
+
+const AONTIMER_BARK: u32 = 0x1;
+const AONTIMER_TIME: u32 = 0x2;
+
+const BARK_INT: u32 = 0;
```

Comment:
Isn't this interrupt # fixed for the chip? If so, shouldn't this be provided by the driver/chip?

Reply:
So the interrupt number here is actually just an index into the interrupts available to the task.  

However, looking at the [original hubris docs](https://hubris.oxide.computer/reference/#_interrupts_from_a_tasks_perspective)
 "When calling irq_control, the task names its interrupts by the notification bits they will set when they fire, rather than their hardware interrupt numbers" 
With this in mind, I think the call to `int_ctrl.enable_int` should probably use the notification bit, not just an index into its interrupts.  This would stay more inline with how interrupts are normally handled.  @FawazTirmizi do you have some context on this?

This change would also make the `BARK_INT` and `AONTIMER_BARK` constants the same, and the shared value would not have to be hard-coded after this [ticket](https://rivosinc.atlassian.net/browse/SW-530) is done.

Reply:
Yes, I don't think `BARK_INT` and `AONTIMER_BARK` need to be two separate constants. Both really seem to be referring to the same thing i.e. notification for bark interrupt.

Reply:
These have been consolidated, but will still need to wait for that ticket to remove the magic numbers.